examples/clojure/extractJarVersion

```diff
@@ -0,0 +1,3 @@
+# Outputs the clojure project version from a passed in .jar file
+version=$(unzip -p $1 META-INF/MANIFEST.MF | grep Leiningen-Project-Version: | sed 's/^.*: //' | tr -d '\r')
+echo $version
```

Comment:
nit: we try to keep all our text files as POSIX files (which require a final newline). It makes it easier to use programs like `cat`.